viewer/packages/cad-geometry-loaders/src/CadManager.ts

```diff
@@ -207,6 +201,7 @@ export class CadManager {
     model.addEventListener('update', this._markNeedsRedrawBound);
     this._cadModelMap.set(model.cadModelIdentifier, model);
     this._cadModelUpdateHandler.addModel(model);
+    this.updateCacheSizeBasedOnBudget(this.budget);
```

Comment:
Can we set the cache size for the added model only, e.g. with a `setCacheSizeForModel` method? I'm not that worried about performance, it just feels somewhat cleaner that way